zk-token-sdk/Cargo.toml

```diff
@@ -33,7 +33,7 @@ bincode = { workspace = true }
 curve25519-dalek = { workspace = true, features = ["serde"] }
 itertools = { workspace = true }
 merlin = { workspace = true }
-rand = { workspace = true }
+rand = "0.8.5"
```

Comment:
And same here for zk-token, please.

Reply:
the code uses curve25519-dalek dependency that uses old API and I think there is some issue with perf regression when that dependency is bumpes...